src/bin/pg_autoctl/ipaddr.c

```diff
@@ -158,19 +160,49 @@ fetchLocalCIDR(const char *localIpAddress, char *localCIDR, int size)
 				break;
 			}
 
+			case AF_INET6:
+			{
+				int i = 0;
+				struct sockaddr_in6 *netmask6
+					= (struct sockaddr_in6 *) ifa->ifa_netmask;
+				struct sockaddr_in6 *address6
+					= (struct sockaddr_in6 *) ifa->ifa_addr;
+
+				struct in6_addr s_network;
+
+				inet_ntop(AF_INET6, (void*) &netmask6->sin6_addr,
+						  netmask, INET6_ADDRSTRLEN);
+
+				inet_ntop(AF_INET6, (void*) &address6->sin6_addr,
+						  address, INET6_ADDRSTRLEN);
+
+				for (i = 0; i < sizeof(struct in6_addr); i++)
+				{
+					s_network.s6_addr[i] =
+						address6->sin6_addr.s6_addr[i]
+						& netmask6->sin6_addr.s6_addr[i];
+				}
+
+				prefix = countSetBitsv6(netmask6->sin6_addr.s6_addr);
+
+				inet_ntop(AF_INET6, &s_network, network, INET6_ADDRSTRLEN);
+
+				break;
+			}
+
 			default:
 				continue;
 		}
 
-		if (strcmp(address, localIpAddress) != 0)
+		if (strcmp(address, localIpAddress) == 0)
 		{
-			continue;
+			found = true;
+			break;
 		}
-		break;
     }
```

Comment:
nit; indenting seems messed up here

Reply:
I don't see it here?

Reply:
I added a commit to fix it, yet again, lets get `citus_indent` setup in the future.